project/Frontend.scala

```diff
@@ -22,6 +22,7 @@ object Frontend extends Build with Prototypes {
       awsSns,
       awsSts,
       awsSqs,
+      cats,
```

Comment:
I can remove cats fairly easily -- it just helped with the way I thought about the problem. Let me know if you would rather I get rid of it